packages/account/src/Components/personal-details/personal-details.jsx

```diff
@@ -83,6 +83,10 @@ const PersonalDetails = ({
         }
 
         errors.document_number = isDocumentNumberValid(document_number, document_type);
+
+        if (document_type.id !== IDV_NOT_APPLICABLE_OPTION.id && !values.confirmation_checkbox) {
+            errors.confirmation_checkbox = 'error';
+        }
```

Comment:
Is there a specific reason we need this check & error related to `confirmation_checkbox`?

Reply:
yes. it helps to handle the state for all fields and safe the status (the value) while switching between document types.
also it will help later to navigate to error field. 
any fields inside the form should be tracked with formik